File: schedule.ios.applicaion/schedule.ios.applicaion/Models/Timetable.swift
//
//  Timetable.swift
//  schedule.ios.application
//
//  Created by Иван Светлов on 11.06.2023.
//

import Foundation

struct Timetable: Identifiable, Codable, Hashable, Equatable {
    var id: Int
    var date: Date
    var groups: [Group]
    var groupNames: String
    var lessons: [Lesson]
}

File: Schedule/Schedule.Core/Models/Timetable.cs
﻿namespace Schedule.Core.Models;

public class Timetable
{
    public int TimetableId { get; set; }

    public int GroupId { get; set; }

    public int DateId { get; set; }

    public virtual Date Date { get; set; } = null!;

    public virtual Group Group { get; set; } = null!;

    public virtual ICollection<Lesson> Lessons { get; set; } = new List<Lesson>();
}